pwndbg/commands/vmmap.py

```diff
@@ -54,12 +55,12 @@ def print_vmmap_gaps_table_header() -> None:
     Prints the table header for the vmmap --gaps command.
     """
     header = (
-        f"{'Start':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}} "
-        f"{'End':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}} "
+        f"{'Start':>{2 + 2 * pwndbg.aglib.arch.ptrsize}} "
+        f"{'End':>{2 + 2 * pwndbg.aglib.arch.ptrsize}} "
         f"{'Perm':>4} "
         f"{'Size':>8} "
         f"{'Note':>9} "
-        f"{'Accumulated Size':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}}"
+        f"{'Accumulated Size':>{2 + 2 * pwndbg.aglib.arch.ptrsize}}"
     )
     print(header)
```

Comment:
This should really be just `print ( ( ...) )` instead of `print(header)`. U can refactor it here or in another PR, whatever